src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/PrisonerDto.kt

```diff
@@ -17,4 +18,7 @@ data class PrisonerDto(
 
   @Schema(description = "prison code", example = "BHI", required = true)
   var prisonCode: String? = null,
+
+  @Schema(description = "Prisoner Alerts", required = true)
+  val alerts: List<PrisonerAlertDto>? = emptyList(),
```

Comment:
Is it optional if it is required and also has an empty list,... I would get rid of ?